src/base/misc/fatfs_boot.S

```diff
@@ -87,9 +87,27 @@ write_msg_20:
 	int	$0x10
 	jmp	write_msg
 
-/*# 5 magic bytes (for IO.SYS compatibility) :-))*/
-	.byte	4, 3, 2, 1, 13
-text:
+	.align 2
+	.globl	boot_txfr
+boot_txfr:
+	.word	0x9090 /* start ofs */
```

Comment:
I don't like to initialise them to zero, it's easier to spot when they've been overwritten or not. Also it helps to trigger a reaction in whatever is reading them. I spotted a problem with MS-DOS 7 when I'd updated the wrong register when converting from the old method that way. That being said I'm happy for them to be zero if you like?